packages/circuits/circom/core/non-qv/processMessages.circom

```diff
@@ -358,14 +338,13 @@ template ProcessOneNonQv(stateTreeDepth, voteOptionTreeDepth) {
 
     // 1. Transform a state leaf and a ballot with a command.
     // The result is a new state leaf, a new ballot, and an isValid signal (0 or 1).
-    var computedNewSlPubKey[2], computedNewBallotNonce, computedIsValid, computedIsStateLeafIndexValid, computedIsVoteOptionIndexValid;
-    (computedNewSlPubKey, computedNewBallotNonce, computedIsValid, computedIsStateLeafIndexValid, computedIsVoteOptionIndexValid) = StateLeafAndBallotTransformerNonQv()(
+    var computedNewSlPubKey[2], computedNewBallotNonce, computedIsValid;
+    (computedNewSlPubKey, computedNewBallotNonce, computedIsValid) = StateLeafAndBallotTransformerNonQv()(
         numSignUps,
         maxVoteOptions,
         [stateLeaf[STATE_LEAF_PUB_X_IDX], stateLeaf[STATE_LEAF_PUB_Y_IDX]],
         stateLeaf[STATE_LEAF_VOICE_CREDIT_BALANCE_IDX],
-        stateLeaf[STATE_LEAF_TIMESTAMP_IDX],
-        pollEndTimestamp,
+        // stateLeaf[STATE_LEAF_TIMESTAMP_IDX],
```

Comment:
why this is commented out?

Reply:
This is an original change from anon joining branch